### PR TITLE
update tgingold/ghdl refs to ghdl/ghdl

### PR DESCRIPTION
**Pull request by 1138-4EB**
Related #500.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -30,6 +37,8 @@ LRM) is a plus.
 
 .. only:: latex
 
+   .. rubric:: 20.12.2017 - GHDL 0.35 was released.
+
```
[the remark this answers is not in the document]
Comment (1138-4EB) [reply doc/index.rst]: Indeed, the idea was to generate a section named 'Changelog' by parsing NEWS.md and generating diferent pages automatically.

Furthermore, I think we should base these 'major changes' on GitHub [milestones](https://help.github.com/articles/about-milestones/). That is. I would not link the 'new' in the docs to github releases (as that is already done in 'Getting GHDL'). Instead, I would link to the corresponding milestone (given that a user-written description is added to the latter).

We are actually using them, but not tightly integrated. In order to do so we have to 'educate' @tgingold to create (dummy) issues (i.e. create and directly close them) to leave track of the motto behind the 'free' commits he contributes. If a given change requires multiple commits, it is ok to create a single issue. But, at now, I feel that the 'changelog' is mostly in his mind.

Anyway, I think that these enhancements are out of the scope of this issue, which is just to make url changes due to the transfer of the repo.

[the remark this answers is not in the document]
Comment (Paebbels) [reply doc/index.rst]: I think for now, a link to the appropriate download would be enough.

The issues are already assigned to milestones by me. I suggest a milestone based on the issue's description. If Tristan closes more or other items, I'm reordering. He is also reassigning issues if my suggestion is wrong. I think this flow is enough for now.

Based on the list of closed issues, it's possible to recreate a changes list.

[the remark this answers is not in the document]
Comment (1138-4EB) [reply doc/index.rst]: > Based on the list of closed issues, it's possible to recreate a changes list..

My comment was not about how issues are handled, which I agree to be ok. I meant the content which is not explicitly reflected in any issue at all, e.g.:

- 'Avoid cascaded error on incorrect type.' and 'Testcase for previous patch.' tgingold committed 6 days ago
- 'hack doc and regenerate texi file.' tgingold committed 7 days ago
- 'simul: handle ...' tgingold committed 9 days ago

> Are you going to add the links; or can I merge?

I added the links. You can merge.

**Comment by 1138-4EB** [comment]
It seems that RTD is not being built since a couple of days, due to a failure. This PR fixed it 7h ago, but it didn't rebuild yet. @tgingold, @Paebbels, can you trigger it manually in http://readthedocs.org/projects/ghdl/builds/?

I am not sure if it is because it stops builing when an error is found, or because we moved the repo and it must be set up again (as happened with travis).

**Comment by Paebbels** [comment]
I changed the repository URL at RTD and setup some keywords (tags). I also disabled branch `new-documentation-structure`. Build was running 3 minutes ago: successful.

**Comment by tgingold** [comment]
Well done!

**Comment by tgingold** [comment]
> We are actually using them, but not tightly integrated. In order to do so we have to 'educate' @tgingold to create (dummy) issues (i.e. create and directly close them) to leave track of the motto behind the 'free' commits he contributes. If a given change requires multiple commits, it is ok to create a single issue. But, at now, I feel that the 'changelog' is mostly in his mind.

I am not sure it is worth. Changelog/news are nice for 'big' features, not for very small ones.
I don't want to spend too much time on non-development work.

**Comment by tgingold** [comment]
> About 'big' and 'small' changes, I agree with you. However, sometimes it is useful to have a complete list of changes, for users to check in case something is not behaving as expected. Say, they are using an invalid syntax that GHDL accepts, but it gets fixed according to the LRM.

Yes, I agree with that.

**Pull request merged**